src/status_im2/contexts/quo_preview/inputs/input.cljs

```diff
@@ -75,11 +74,13 @@
                              :on-char-limit-reach #(js/alert
                                                     (str "Char limit reached: " %))})]
     (fn []
-      (let [background-color (case (:variant @state)
-                               :dark-blur  "rgb(39, 61, 81)"
-                               :dark       colors/neutral-95
-                               :light-blur "rgb(233,247,247)"
-                               :white)
+      (let [background-color (if (:blur @state)
```

Comment:
@smohamedjavid added a blur background you can use, check it out in the preview screens.  For status-tags component and profile input component
they're using a component preview/blur-view 👍

Reply:
I'll look at it! thanks! :+1:

Reply:
That component makes things easier! I added it!